server/routes/bookVisit/additionalSupportController.ts

```diff
@@ -1,16 +1,15 @@
 import type { RequestHandler } from 'express'
 import { ValidationChain, body, validationResult } from 'express-validator'
 
-export default class SelectAdditionalSupportController {
+export default class AdditionalSupportController {
   public constructor() {}
 
   public view(): RequestHandler {
     return async (req, res) => {
-      res.render('pages/bookingJourney/selectAdditionalSupport', {
+      res.render('pages/bookVisit/additionalSupport', {
         errors: req.flash('errors'),
         formValues: req.flash('formValues')?.[0] || {},
-        booker: req.session.booker,
-        bookingJourney: req.session.bookingJourney,
+        prisonName: req.session.bookingJourney.prison.prisonName,
```

Comment:
Unrelated but spotted we were passing unnecessary data to template: only prison name actually used.